apps/api/src/app/channels/channels.controller.ts

```diff
@@ -1,70 +0,0 @@
-import { Body, ClassSerializerInterceptor, Controller, Put, UseGuards, UseInterceptors } from '@nestjs/common';
-import { IJwtPayload, MemberRoleEnum } from '@novu/shared';
-import { UserSession } from '../shared/framework/user.decorator';
-import { JwtAuthGuard } from '../auth/framework/auth.guard';
-import { Roles } from '../auth/framework/roles.decorator';
-import { UpdateMailSettings } from './usecases/update-mail-settings/update-mail-settings.usecase';
-import { UpdateMailSettingsCommand } from './usecases/update-mail-settings/update-mail-settings.command';
-import { UpdateSmsSettings } from './usecases/update-sms-settings/update-sms-settings.usecase';
-import { UpdateSmsSettingsCommand } from './usecases/update-sms-settings/update-sms-settings.command';
-import { ApiExcludeController, ApiOperation, ApiTags } from '@nestjs/swagger';
-import { UpdateMailSettingsRequestDto } from './dtos/update-mail-settings-request.dto';
-import { UpdateSmsSettingsRequestDto } from './dtos/update-sms-settings-request.dto';
-import { UpdateSettingsResponseDto } from './dtos/update-settings-response.dto';
-import { ExternalApiAccessible } from '../auth/framework/external-api.decorator';
-import { ApiResponse } from '../shared/framework/response.decorator';
-
-@Controller('/channels')
```

Comment:
what about this controller and endpoints?

Reply:
This was an old controller me and @scopsy said we now could remove because there is no endpoints left in it that is used.